frontend/rust-lib/flowy-folder/src/manager.rs

```diff
@@ -1182,24 +1209,25 @@ impl FolderManager {
 
     Some(view)
   }
+
   async fn get_publish_payload(
```

Comment:
it would be better to yield at some point if this task is a cpu bound or using spawn_blocking instaed

for example, If you don't use spawn_blocking, you need to yield by using `tokio::task::yield_now().await;` Otherwise, you might stuck the runtime